addons/mass_mailing_hr/static/src/fields/html_field/mass_mailing_html_field.js

```diff
@@ -0,0 +1,28 @@
+import { patch } from "@web/core/utils/patch";
+import { MassMailingHtmlField } from "@mass_mailing/fields/html_field/mass_mailing_html_field";
+
+patch(MassMailingHtmlField.prototype, {
+    getThemeSelectorConfig() {
+        return {
+            ...super.getThemeSelectorConfig(),
+            /**
+             * If the selected theme is Basic (text-only) and the current model is employee:
+             * Remove the unsubscribe element, as employees shouldn't blacklist themselves from your mailings
+             */
+            setThemeHTML: (html) => {
+                if (this.props.record.data.mailing_model_name == "hr.employee") {
+                    const transferEl = document.createElement("div");
+                    transferEl.innerHTML = html;
+                    if (
+                        transferEl.querySelector(".o_basic_theme") &&
+                        transferEl.querySelector(".unsubscribe_link")
+                    ) {
+                        transferEl.querySelector(".unsubscribe_link").remove();
```

Comment:
This will modify the theme permanently even if the user change recipient model afterwards. You should instead create a normal editor plugin that would be added in the basic editor, and remove the link before start_edition (i.e. during the Plugin setup).

You can just add the plugin in the `basic-editor-plugins` registry, and it will be loaded for the basic editor.